liquibase-standard/src/main/java/liquibase/ui/UIServiceEnum.java

```diff
@@ -0,0 +1,17 @@
+package liquibase.ui;
+
+public enum UIServiceEnum {
```

Comment:
Please format this file, it's indented wrong.

Also, please add some javadoc.

Reply:
+ lombok